smarts/sstudio/genhistories.py

```diff
@@ -117,6 +128,14 @@ def _create_tables(self, dbconxn):
                    FOREIGN KEY (vehicle_id) REFERENCES Vehicles(id)
                ) WITHOUT ROWID"""
         )
+        ccur.execute(
+            """CREATE TABLE Traffic_Lights (
+                   sim_time REAL NOT NULL,
```

Comment:
I think we're going to want an index on the `sim_tiime` column given that this is queried on every step (potentially for multiple vehicles).